src/pages/lib-v1/ssr/nextjs/q/platform/[platform].mdx

```diff
@@ -0,0 +1,8 @@
+export const meta = {
+  title: `Use Amplify with Next.js`,
+  description: `Use Amplify with Next.js`,
+};
+
+import js0 from "/src/fragments/lib-v1/ssr/nextjs/getting-started.mdx";
```

Comment:
Let's eliminate pages that just have one fragment. Move the fragment content directly inline into the page.

Reply:
Cleaned this up here and in other pages we touched.